doc/xml/flow.txt

```diff
@@ -0,0 +1,52 @@
+graph TB
+    IO_READ_INTERNAL(("ioReadInternal()")) --> DRIVER_EOF
```

Comment:
Let's use the start/end symbols for line 2:
`IO_READ_INTERNAL(-"ioReadInternal()"-) --> DRIVER_EOF`

and line 52:
`RETURN(-Return-)`